packages/cascara/src/modules/DataCheckbox/DataCheckbox.js

```diff
@@ -61,9 +62,9 @@ const DataCheckbox = ({
   // Do not render an editable input if the module is not editable
   return (
     <ModuleErrorBoundary>
-      <div className={styles.Checkbox}>
+      <Role className={styles.Checkbox}>
```

Comment:
@rociobaena everything in `cascara/src/modules` should _not_ be using Role. These are not exported, root level elements. All of the modules should be returned back to the `div`. Remember... this is only for exported, root-level components so we can use `as` on them.